spec/rspec/support/ruby_features_spec.rb

```diff
@@ -144,6 +145,20 @@ def ripper_can_parse_source_including_keywordish_symbol?
           end
         end
 
+        # https://github.com/jruby/jruby/issues/5209
+        def ripper_can_parse_source_referencing_keyword_arguments?
+          in_sub_process_if_possible do
+            require 'ripper'
+            begin
+              # It doesn't matter if MRI compatibility is < 2.
+              Gem::Version.new(RUBY_VERSION) < Gem::Version.new("2.0") ||
```

Comment:
:wave: You can't use `Gem::Version` within RSpec as we disable rubygems, I suspect this is why the build is failing. String comparison works fine for our purposes.

Reply:
Thanks! I'd already switched to string comparison before I saw your message, but it's good to know what went wrong.